src/test/java/teammates/test/cases/common/StringHelperTest.java

```diff
@@ -248,6 +250,17 @@ public void testTruncateHead() {
         assertEquals("123456789", StringHelper.truncateHead("123456789", 10));
         assertEquals("567890", StringHelper.truncateHead("1234567890", 6));
     }
+    
+    @Test
+    public void testRemoveEnclosingSquareBrackets() {
+        String list = Arrays.asList(new String[]{"test1", "test2"}).toString();
+        String nullString = null;
```

Comment:
Probably not needed either. If needed, try to define the variable close to where it is needed.